<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdMan</class>
 <widget class="QWidget" name="ThresholdMan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QPushButton" name="pushOK">
     <property name="text">
      <string>&amp;Create new</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushApply">
     <property name="text">
      <string>&amp;Apply in place</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Thresholding options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="4">
       <widget class="QRadioButton" name="radioOtsuSPThres">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Otsu spa&amp;cial</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Lower threshold: </string>
        </property>
        <property name="buddy">
         <cstring>horizontalSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="radioOtsuThres">
        <property name="text">
         <string>Ots&amp;u method</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QRadioButton" name="radioTriangleThres">
        <property name="text">
         <string>Tr&amp;iangle</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioManualThres">
        <property name="text">
         <string>&amp;Manual </string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushCancel">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>320</height>
      </size>
     </property>
     <property name="title">
      <string>Histogram</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4"/>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Histogram options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Cut o&amp;ff: </string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBox_2</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QComboBox" name="appMethod">
        <item>
         <property name="text">
          <string extracomment="Apply current threshold on complete dataset">2D thres - 3D application</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string extracomment="Apply threshold on current slide only. ">2D thres - 2D appl (one)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2D thres - 2D appl (recr)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string extracomment="Apply current threshold to all slides">2D thres - 2D app (all)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3D thres</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="value">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bin Si&amp;ze:</string>
        </property>
        <property name="buddy">
         <cstring>spinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Applicatio&amp;n method:</string>
        </property>
        <property name="buddy">
         <cstring>appMethod</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="value">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkMask">
        <property name="text">
         <string>Create Mask</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkGlobal">
        <property name="text">
         <string>3D Histogramming</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionThresholdMethodChanged">
   <property name="text">
    <string>ThresholdMethodChanged</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioManualThres</sender>
   <signal>clicked()</signal>
   <receiver>actionThresholdMethodChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioTriangleThres</sender>
   <signal>clicked()</signal>
   <receiver>actionThresholdMethodChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioOtsuThres</sender>
   <signal>clicked()</signal>
   <receiver>actionThresholdMethodChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
